src/MainWindow.vala

```diff
@@ -65,5 +79,15 @@ public class ShortcutOverlay.MainWindow : Gtk.Window {
 
 
         add (layout);
+        get_style_context ().add_class ("rounded");
+        set_titlebar (headerbar);
+
+        settings_button.clicked.connect (() => {
+            try {
+                AppInfo.launch_default_for_uri ("settings://input/keyboard/shortcuts", null);
+            } catch (Error e) {
+                warning ("%s\n", e.message);
```

Comment:
Should just use `warning (e.message)` as warning takes care of the formatting alone.